tsconfig.json

```diff
@@ -1,21 +1,18 @@
 {
   "compilerOptions": {
     "moduleResolution": "node",
-    "target": "es5",
-    "module":"es6",
-    "lib": ["es2015", "es2016", "es2017", "dom"],
+    "target": "ES2020",
+    "module":"ESNext",
```

Comment:
What's the reason to prefer this over the previously used values?

Reply:
I want the files in `lib/` to not be transpiled for older versions of node as little as possilbe.
The reasoning for that is: (1) Because of webcrypto usage, we require node >=15 (node >= 16 in terms of LTS) anyway, and that supports ES2020 and ESNext modules, and (2) because applications will either use our pre-bundled `index.min.js` file, or they'll bundle the code (and therefore transpile it to older versions of js) in a way they need for their use-case.

Practically this means that for applications that don't need to support, let's say "pretty old firefox", we get reduced bundle sizes and slightly faster performance.